packages/sit-onyx/src/components/TestInput/TestInput.vue

```diff
@@ -8,10 +8,22 @@ const props = withDefaults(
     /** Label to show next to the input */
     label?: string;
     required?: boolean;
+    minlength?: number;
+    maxlength?: number;
+    min?: number;
+    max?: number;
+    type?: string;
+    pattern?: string;
   }>(),
   {
     modelValue: "",
     label: "",
+    minlength: undefined,
+    maxlength: undefined,
+    min: undefined,
+    max: undefined,
+    type: undefined,
+    pattern: undefined,
```

Comment:
This is caused by eslint, right?^^

Reply:
yep, eslint wants to have everything defined in the default :( I planned to discuss this with you guys once the PR is ready.

Reply:
Partial helped :)